esphomeyaml/components/my9231.rst

```diff
@@ -0,0 +1,59 @@
+MY9231/MY9291 LED driver Component
+==================================
+
+The MY9231/MY9291 component represents a MY9231/MY9291 LED diver chain
+(`MY9231 description <http://www.my-semi.com.tw/file/MY9231_BF_0.91.pdf>`__,
+`MY9291 description <http://www.my-semi.com.tw/file/MY9291_BF_0.91.pdf>`__) in
+esphomelib. Communication is done with two GPIO pins (DI and DCKI) and multiple
+driver chips can be chained. There are two models with different number of
+output channels (MY9291 with 4 channels and MY9231 with 3 channels). They are
+popular driver chips used in smart light blubs:
+
+- Sonoff B1 (MY9231)
```

Comment:
If you've tested this with the Sonoff B1, could you add a section after `Configuration Variables` and `See Also` with an example config for Sonoff B1s?

Otherwise people will keep asking me how they can set up their Sonoff B1 using this integration and I have to tell them I don't know because I don't own one of them :P

Reply:
After `See Also`? Or between `Configuration Variables` and `See Also`? I did not find any entry with a section after `See Also`.

Reply:
Sorry, I meant between the two

Reply:
Example configuration added.